quipucords/api/messages.py

```diff
@@ -81,7 +81,10 @@
 # scan jobs messages
 PLURAL_SCAN_JOBS_MSG = 'Scan Jobs'
 SJ_REQ_SOURCES = 'Scan job must have one or more sources.'
+SJ_SCAN_IDS_INV = 'Scan identifiers must be integer values.'
```

Comment:
Scan job identifiers ...

Reply:
These are right.  SJ_ prefix is for messages in the ScanJob classes and its referencing the Scan not the Scan job identifier.